### ISSUE TITLE
The Bot response time "A minute ago" is not announced by screen reader NVDA in both Focus and Browse mode.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5015]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Visually challenged users who rely on screen readers will be impacted if the bot response time is not announced by screen reader NVDA in both Focus and Browse mode.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Others or unrelated

### What is the public URL for the website?

https://airtuserexpdev.azurewebsites.net/Dashboard 

### How to reproduce the issue?

Step 1: Open the URL: [https://airtuserexpdev.azurewebsites.net/Dashboard](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fairtuserexpdev.azurewebsites.net%2fDashboard&data=05%7c01%7cv-shravyar%40microsoft.com%7c09aeb38517334354d48308dbeca14400%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638363949604189146%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=X9RebjeUY64a0z9YqF59KH8oRldy59wbtdE%2fzj0CjvM%3D&reserved=0) in Edge Browser.

Step 2: AIRT Dashboard page will appear, Navigate to “Chat with Sage” chat box at the bottom of the page.

Step 3: Enable browse mode and navigate to "A minute ago" using down arrow key.

Step 4: Verify whether the Bot response time "A minute ago" is announcing by screen reader NVDA in both Focus and Browse mode or not.

### What do you expect?

The Bot response time "A minute ago" should be announced by screen reader NVDA.
NVDA should announce as "A minute ago" upon focusing on the "A minute ago" text in browse mode.

Note: Issue repro with all screen readers

### What actually happened?

The Bot response time "A minute ago" is not announced by screen reader NVDA in both Focus and Browse mode.

### Do you have any screenshots or recordings to repro the issue?

![MAS 1 3 1_Bot response time  is not announced by both focus and browse mode](https://github.com/microsoft/BotFramework-WebChat/assets/117266902/95751af5-2ffe-481a-8c9d-3903f329c063)

### MAS reference

MAS 1.3.1 – Info and Relationships

**Comment by Kirank915** [comment]
Bug copied from Accenture team: [BUG 11645889](https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/11645889)

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS1.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by compulim** [comment]
We discussed with accessibility team and they suggested narrating an absolute timestamp, rather than a relative timestamp. This is because the relative timestamp will change from time to time.

Instead of narrating "a minute ago", we narrate the time, for example "three twenty P-M" for (3:20 PM).
